src/decision/dMCDA.jl

```diff
@@ -412,40 +432,41 @@ function create_shade_matrix(A, max_area, wt_conn_shade, wt_waves, wt_heat, wt_p
     wsh .= mcda_normalize(wsh)
```

Comment:
What was the reasoning behind the normalization again?

It was removed for seeding.

Reply:
The set of weights should be normalised. It looks like you removed the seeding weight normalisation 6 months ago because of duplication, but maybe the original was also removed. I just checked JMcDM though and it looks like JMcDM does normalisation for weights and matrices internally also, so we can get rid of matrix and weights normalisation.

Reply:
Sorry, correction- we can remove normalisation when we use JMcDM methods. So I'll move it up to only be used if a non-JMcDM method is selected.